plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-name/frontend.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * External dependencies
+ */
+import { store } from '@woocommerce/interactivity';
```

Comment:
This one needs to be updated too:

```suggestion
import { store } from '@wordpress/interactivity';
```